src/Specular/Ref.purs

```diff
@@ -3,24 +3,27 @@ module Specular.Ref
   , newRef
   , new
   , newRefWithEvent
-  , refValue
   , value
-  , refUpdate
   , modify
   , refUpdateConst
   , set
   , focusRef
   , pureFocusRef
   , previewRef
-  , updateRef
   , readRef
 
   , constRef
 
   , wrapViewWidget
-
+  , Callback
```

Comment:
I'd remove even the type alias, and just use `a -> Effect Unit`.

Reply:
Wow, that was _fast_ @kozak  :)

Reply:
Done